woocommerce/woocommerce-android
# Issue/5711 interac refund navigation logic

part of: #5711 


### Description

This PR adds code that helps in navigating to the card reader screen from the refund screen for the Interac refund.

### Testing instructions

Since this PR just adds a supporting logic required for full fledge Interac refund navigation and there are some missing parts in this PR which are handled in the next PR. Green CI should be enough. In order to see what the end product looks like, please refer to [this](https://github.com/woocommerce/woocommerce-android/pull/6157) PR.

### Images/gif
